src/components/probe/index.ts

```diff
@@ -319,6 +319,36 @@ export async function doProbe({
       }
     }
 
+    if (probe?.mysql) {
```

Comment:
Should this be `probe?.mysql || probe?.mariadb?`

Reply:
good call.. done